_posts/en/newsletters/2021-12-01-newsletter.md

```diff
@@ -0,0 +1,99 @@
+title: 'Bitcoin Optech Newsletter #177'
+permalink: /en/newsletters/2021/12/01/
+name: 2021-12-01-newsletter
+slug: 2021-12-01-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter describes a recently fixed interoperability issue
+between different LN software and includes our regular sections with a
+list of new releases and release candidates plus notable changes to
+popular Bitcoin infrastructure software.
+
+## News
+
+- **LN interoperability:** a recent change to the LN specification
+  described in [Newsletter #165][news165 bolts880] was implemented in different ways by
+  different LN nodes, resulting in nodes running the latest version of
+  LND being unable to open new channels with nodes running the latest
+  versions of C-Lightning and Eclair.  LND users are encouraged to
+  upgrade to a bug fix release, 0.14.1 (described in the *release*
+  section below).
+
+    A related [discussion][xraid interop] was started on the
+    Lightning-Dev mailing list about improving interoperability testing.
+    Christian Decker, who previously created and maintained the LN
+    [integration testing framework][ln integration], said that [he
+    believes][decker interop] basic interoperability testing is
+    "unlikely to catch any but the most egregious issues".  LN
+    developers participating in the discussion [noted][zmn interop] that
+    catching these sort of bugs is why every major implementation
+    provides release candidates (RCs) and why expert users and
+    administrators of production systems are encouraged to contribute to
+    the testing of those RCs.
+
+    For anyone interested in participating in such testing, the Optech
+    newsletter lists RCs from four major LN implementations, plus
+    various other Bitcoin software.  The LND version under discussion was
+    listed as an RC in Newsletters [#174][news174 lnd] and [#175][news175
+    lnd].
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [LND 0.14.1-beta][] is a maintenance release that fixes the issue
+  described in more detail both in the *news* section above and in the
+  description of [LND #6026][] in the *notable changes* section below.
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[Rust-Lightning][rust-lightning repo], [libsecp256k1][libsecp256k1
+repo], [Hardware Wallet Interface (HWI)][hwi repo],
+[Rust Bitcoin][rust bitcoin repo], [BTCPay Server][btcpay server repo],
+[BDK][bdk repo], [Bitcoin Improvement Proposals (BIPs)][bips repo], and
+[Lightning BOLTs][bolts repo].*
+
+- [Bitcoin Core #16807][] Updates address validation to return the indices of
+  likely invalid characters in [Bech32 and Bech32m][topic bech32] addresses.
```

Comment:
Gonna s/invalid characters/typos/ since "invalid characters" seems to imply only characters that are fundamentally invalid (e.g. not part of the bech32 alphabet) whereas "typo" feels more general to me.

Reply:
For some reason we published "s/invalid characters/typos characters/" instead. I've opened a PR here: https://github.com/bitcoinops/bitcoinops.github.io/pull/695